crates/db_schema/src/source/comment.rs

```diff
@@ -51,6 +51,17 @@ pub struct Comment {
   /// Whether the comment has been distinguished(speaking officially) by a mod.
   pub distinguished: bool,
   pub language_id: LanguageId,
+  pub score: i64,
```

Comment:
I was wondering if we should add any comments to denote that some fields are built by triggers / reports, but I guess its kind of arbitrary.